themes/theme-docs/src/components/Dialog.styles.ts

```diff
@@ -1,7 +1,7 @@
 import { ThemeComponent, cva } from '@marigold/system';
 
 export const Dialog: ThemeComponent<'Dialog'> = {
-  closeButton: cva('absolute right-2 top-2', {
+  closeButton: cva('', {
```

Comment:
I think you have to add this again :)

Reply:
I added it directly on CloseButton component @sarahgm

Reply:
<img width="321" alt="Bildschirmfoto 2024-10-21 um 13 33 24" src="https://github.com/user-attachments/assets/f5ce554c-7f2b-426b-9265-80836d3ba46a">
but this doesn't look right to me? :O

Reply:
I think we said the button should align with the headline